src/styles/container.scss

```
@@ -0,0 +1,45 @@
+.container {
+  display: grid;
+  grid-template-areas: "first"

```

```suggestion
  grid-template-areas: "first"

```
As a variant you could use something `like grid-template-columns: repeat(2, 1fr);` etc